platform/android/src/unaccent.cpp

```diff
@@ -0,0 +1,17 @@
+#include <unaccent.hpp>
+#include <string>
+#include <src/java/lang.hpp>
+#include "attach_env.hpp"
+#include "text/collator_jni.hpp"
+
+namespace mbgl {
+namespace platform {
+
+std::string unaccent(const std::string& str) {
+    android::UniqueEnv env = android::AttachEnv();
+    jni::String unaccented = android::StringUtils::unaccent(*env,jni::Make<jni::String>(*env, str));
```

Comment:
Do we need to delete refs here to avoid table overflows in tight loops?

Reply:
yes, looking forward using `jni::SeizeLocal` to avoid these issues!

Reply:
It's going to be even better than that -- https://github.com/mapbox/jni.hpp/pull/40.